game-headed/src/main/java/org/triplea/game/client/ui/javafx/TripleA.java

```diff
@@ -62,6 +70,7 @@ private void setupStage(final Stage stage, final Scene scene) {
     stage.getIcons().add(new Image(getClass().getResourceAsStream(FxmlManager.ICON_LOCATION.toString())));
     stage.setTitle("TripleA");
     stage.show();
+    stage.setOnCloseRequest(e -> exit());
```

Comment:
Are we sure we want to exit on close?

Reply:
Well this code is put in place so that when pressing Alt+F4 the EDT is terminated properly if no windows remain open, so yes.